🐛(objects) make objects name unique

jmaupetit opened the pull request:
## Purpose

When two (or more) objects from different applications share the same name, objects conflict with each others event if their labels differ.

## Proposal

- [x] Prefix route by their related application name to avoid cross-apps collisions in route names
- [x] Prefix all applications objects by their application name

jmaupetit committed:
🐛(route) make route name unique

Prefix route by their related application name to avoid cross-apps
collisions in route names.

jmaupetit commented:
It seems that the collision occurs for every OS objects, not only routes. I'll update this PR description.

jmaupetit committed:
🐛(apps) make applications object names unique

All OS objects should have a unique name, hence we must prefix them with
the application name to avoid collisions between applications services.

sampaccoud commented on apps/richie/templates/richie/dc.yml.j2:
```diff
@@ -6,20 +6,20 @@ metadata:
     version: "{{ richie_image_tag }}"
     deployment_stamp: "{{ deployment_stamp }}"
-  name: richie-{{ deployment_stamp }}
+  name: "richie-richie-{{ deployment_stamp }}"
```
Arf... Remember when I told you that the service should be called "app" and not richie. I think you really need to consider that. And it's more natural to follow the docker-compose semantic...

jmaupetit replied:
Ok, you are right: let's do this.

jmaupetit committed:
🚚(richie) move richie service name to app

The "richie" service from Richie's application is now called "app" to
fit with our docker-compose naming scheme.
